src/cart/crt_init.c

```diff
@@ -29,7 +29,9 @@ dump_envariables(void)
 		"OFI_PORT", "OFI_INTERFACE", "OFI_DOMAIN", "CRT_CREDIT_EP_CTX",
 		"CRT_CTX_SHARE_ADDR", "CRT_CTX_NUM", "D_FI_CONFIG",
 		"FI_UNIVERSE_SIZE", "CRT_ENABLE_MEM_PIN",
-		"FI_OFI_RXM_USE_SRX", "D_LOG_FLUSH", "CRT_MRC_ENABLE" };
+		"FI_OFI_RXM_USE_SRX", "D_LOG_FLUSH", "CRT_MRC_ENABLE",
+		"RDMAV_HUGEPAGES_SAFE", "UCX_LOG_LEVEL", "UCX_MEM_LOG_LEVEL",
+		"UCX_IB_FORK_INIT" };
```

Comment:
Same here as other comment, I would separate UCX from OFI env variables that we dump instead of having that huge list of env vars. If you think that should be done in a separate PR, it would be good to enter an issue for it.

Reply:
I removed all UCX environment variables from the list for this PR.  I agree with the suggestion to do the refactor in a subsequent PR.